modules/apps/headless/headless-batch-engine/headless-batch-engine-impl/src/main/java/com/liferay/headless/batch/engine/internal/resource/v1_0/util/ParametersUtil.java

```diff
@@ -26,8 +26,6 @@ public static Map<String, Serializable> toParameters(
 		UriInfo contextUriInfo, Set<String> ignoredParameters) {
 
 		return HashMapBuilder.<String, Serializable>putAll(
-			_toMap(ignoredParameters, contextUriInfo.getPathParameters())
```

Comment:
We are removing the path parameters. Are we sure we are doing the right thing?

See the path parameters:

https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/headless/headless-batch-engine/headless-batch-engine-impl/src/main/java/com/liferay/headless/batch/engine/internal/resource/v1_0/BaseExportTaskResourceImpl.java#L164